app/Dockerfile

```diff
@@ -1,13 +1,22 @@
 # syntax=docker/dockerfile:1
+# Download and install python dependencies in a container
+FROM python:3.8.17 as dependency-install-container
```

Comment:
If you're doing Docker, you might as well bump this up to 3.10 at least...

Reply:
Created #840